lib/shared/routes/routers.dart

```diff
@@ -148,6 +149,9 @@ final _customerAppRouter = NavRouter(
           fullscreenDialog: true,
         );
         break;
+      case CustomerAppRoutes.review:
```

Comment:
You were supposed to implement this the way other pages like CustomerAppRoutes.artisanSwitch and CustomerAppRoutes.editPrimary were done and not like CustomerAppRoutes.pickService.